packages/back-end/src/controllers/experiments.ts

```diff
@@ -339,7 +339,7 @@ async function _getSnapshot(
   );
 }
 
-async function _getSnapshots(
+export async function _getSnapshots(
```

Comment:
Move this outside of the controller.  We have some code that expects all exported methods of a controller file to be request handlers with the format `(req, res) => Promise<void>`